mixer/pkg/runtime2/handler/table.go

```diff
@@ -135,10 +133,10 @@ func (t *Table) Cleanup(current *Table) {
 			err = panicErr
 		}
 
+		_ = entry.env.ensureWorkerClosed()
```

Comment:
Why is this result being ignored?

Reply:
The only error of importance seems to be not being able to get the counter. I think we can get the error and log the error at this level, instead of logging to env itself.